backend/api/tests/views/test_utils.py

```diff
@@ -56,7 +56,7 @@ def test_download_file_local_allowed(self):
         self.assertIn("local-organization", metadata.permissions_process_authorized_ids)
 
         with mock.patch("api.views.utils.get_owner", return_value="local-organization"):
-            response = self.client.get(self.function_url, **self.extra)
+            response = self.client.get(self.function_url)
```

Comment:
I believe this one is needed since here the client is an OrganizationUser not an AuthenticatedUser 
So to me, two solutions, either I leave the extra for this kind of client (there are other extra elsewhere in the code) or I also change this kind of client to have a default channel (but I'm not sure it makes sense, I need to think about this)

Reply:
Sorry I didn't see before the other use of `self.extra`